iceoryx_hoofs/source/posix_wrapper/semaphore.cpp

```diff
@@ -190,7 +188,7 @@ Semaphore::Semaphore(CreateUnnamedSharedMemorySemaphore_t, const unsigned int va
 Semaphore::Semaphore(OpenNamedSemaphore_t, const char* name, const int oflag) noexcept
     : m_isCreated(false)
 {
-    if (m_name.unsafe_assign(name) == false)
+    if (!m_name.unsafe_assign(name))
```

Comment:
are we fine with `!` instead of `== false` as suggested by `readability-simplify-boolean-expr`?